docs/post-install.md

```diff
@@ -0,0 +1,61 @@
+==> valkyrie: ok: [localhost] => {
+==> valkyrie:     "var": {
+==> valkyrie:         "ssh_public_key_output.stdout_lines": [
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////",
+==> valkyrie:             "  Add the following deploy key to your project on Github:",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCosL1qmesmA55S2ZB3fEdsLQSElHOARcXpu7ZeT5vX+kpKb6OQQ/6zQ9I5nx1GaZORkwlKy4qNxhmUiy7u/aIiPALQlL2m9y7brqFxewPMCcX5vhV84rW/qxieJzx3nSC0s99ui+vnYhylbwBapJmfpUcuCMJzyP+JlusRYTLyk/uIrDtewnk2PgAtFYF/gUyK6It/RIbwdtawXrW/MQuFuYPpVYnvG+bbqmx+t7HBISWBbm/DK2guKhXzkUhdD24sdPp9tjUfLfsWYB3Plh5mhOM8I4cTwjVOxGwVfwC+KiEAxlRbSihsqUaRdIr7AXfZ4pG/b6MelwlcX3W61tvz Valkyrie deploy key",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////"
+==> valkyrie:         ]
+==> valkyrie:     }
+==> valkyrie: }
+==> valkyrie: 
+==> valkyrie: TASK: [getvalkyrie.valkyrie | Output the login link] ************************** 
+==> valkyrie: ok: [localhost] => {
+==> valkyrie:     "var": {
+==> valkyrie:         "aegir_login_link_output.stdout_lines": [
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////",
+==> valkyrie:             "  You can access the front-end of your Aegir install at:",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "http://valkyrie.local/user/reset/1/1429111863/uAvC9VJx-l2CqhaufWg8ElCSFWzfdTiUTxJOQJ-pbz4/login",
+==> valkyrie:             "=============================================================================",
+==> valkyrie:             "/////////////////////////////////////////////////////////////////////////////"
+==> valkyrie:         ]
+==> valkyrie:     }
+==> valkyrie: }
+```
+
+* Use the login link as a one-time access to the now deployed aegir frontend, and reset the admin password immediate.  Don't forget to tick the check boxes for all the roles you would want the admin user to be able to play on the aegir server.
+* Copy the displayed deploy ssh key, and add it to your project on github.  This will allow you to publish to the project from valkyrie.
+
+### Looking Around the system
+
+**The directory in which vagrant commands are executed is significant**
+* The default location for vagrant base boxes is ~/.vagrant/machine, but valkyrie does not use that
+* The base directory for valkyrie vagrant vm is the directory in which you invoked the command `drush vmnew valkyrie`
+* To see all vagrant instances for your user account, from anywhere type `vagrant global-status` .. the display of this command is cached and might show recently deleted VMs
```

Comment:
everything below here is standard Vagrant functionality. I don't think there's any reason to describe it in detail here.